frontend/src/utils/Copy.ts

```diff
@@ -145,3 +145,6 @@ export const CONFIRM_OK = "Okay";
 
 export const USER_WANTS_REVIEWER =
   "This user is requesting to be evaluated for the reviewer role";
+
+export const SIGN_UP_ALERT_PASSWORD_LESS_THAN_6_CHARACTERS =
+  "For security, please consider a longer password.";
```

Comment:
nit: firebase actually throws an error if the password is shorter than 6 chars. maybe copy like this is more clear:
`Please enter a password at least 6 characters long.`